src/Agent/NewRelic/Agent/Core/DataTransport/SpanStreamingService.cs

```diff
@@ -25,9 +25,15 @@ public SpanStreamingService(IGrpcWrapper<Span, RecordStatus> grpcWrapper, IDelay
 
         protected override void HandleServerResponse(RecordStatus responseModel, int consumerId)
         {
+            if (responseModel.MessagesSeen == 0)
```

Comment:
Is the log message that this if block prevents from happening still useful when MessagesSeen is 0? If we still have it, it could serve as a heartbeat to let us know that things are still connected. With that being said, an application with a steady load should almost always see a non-zero value here.